contracts/state.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity 0.8.19;
+
+// SPDX-License-Identifier: MIT
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+struct State {
+    address payable owner;
+    address payable intermediary;
+    uint turnNum;
+    uint intermediaryBalance;
```

Comment:
It could be nice to also include the `ownerBalance`. That way you can determine the owner balance by just inspecting the `State` instead of having to also query the nitro SCW balance.